packages/binding.template/spec/templatingBehaviors.ts

```diff
@@ -539,10 +539,14 @@ describe('Templating', function () {
         return [1, 3, 8]
       }
 
-      override nodeHasBindings(node: Element, bindingContext?: BindingContext) {
-        return node.tagName == 'EM' || originalBindingProvider.nodeHasBindings(node, bindingContext)
+      override nodeHasBindings(node: Node, bindingContext?: BindingContext) {
+        if (node instanceof Element)
+          return node.tagName == 'EM' || originalBindingProvider.nodeHasBindings(node, bindingContext)
```

Comment:
if (condition) followed by newline is dangerous.  Better to always use braces.  Not a blocker, but noting.